lib/slack/state.ex

```diff
@@ -86,6 +86,11 @@ defmodule Slack.State do
     put_in(slack, [:users, user, :presence], presence)
   end
 
+  def update(%{type: "presence_change", users: users, presence: presence}, slack) do
+    users
+    |> Enum.reduce(slack, fn(user, acc) -> put_in(acc, [:users, user, :presence], presence) end)
```

Comment:
can you split this across multiple lines instead of 1lining it?

Reply:
Done.